org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -206,6 +218,38 @@ public IProxyService getProxyService() {
 		return null;
 	}
 
```

Comment:
@aeschli , seems language server need these info, too. ref [the feature](https://github.com/eclipse/eclipse.jdt.ls/issues/48)